### ISSUE TITLE
[BUG] Protocol 2 only works if no check mark is set

### ISSUE THREAD

**Issue by DH1KLM** [sjk7/Thetis-2.9.0#5]
Protocol 2 only works for me if no check mark is set.
That's a bit misleading.

WIN10, ANAN7000, protocol 2.

![net](https://user-images.githubusercontent.com/27503182/232206578-7a256970-f7ae-4266-a187-39b609f0812b.JPG)

**Comment by sjk7** [comment]
Hello! As mad as it sounds, this is by design (not my doing). The checkbox has three states: Checked, unchecked and intermediate. It just so happens the P2 setting is the "unchecked" state, so all is well.  Confused? ;-)

This is not my doing, and I don't like the UI design, but that's the way it is currently. I did just upload some new installers in Releases. There are/have been other issues I have tried to address in the new build (There was some funk with the Protocols/sample rates).
